cloudsim-plus-testbeds/src/main/resources/SlaCustomer1.json

```diff
@@ -0,0 +1,88 @@
+{
+  "metrics": [
+    {
+      "name": "TaskTimeCompletion",
+      "dimensions": [
+        {
+          "name": "valueMin",
```

Comment:
The dimension name "valueMin" and "valueMax" is incorrect. It should be "minValue" and "maxValue".
Change the name and the values of the constants inside the SlaMetricDimension class
and the name of the methods isValueMax and isValueMin.

You have to change inside all these SlaCustomer and the SlaMetrics JSON files.
You can right click inside the directory where the SLA files are in IntelliJ and choose "Replace in path" to change all JSON files.